### PR TITLE
Add variants in sidebar

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard14**
Add the variants page in the sidebar, where it finds the highest variant and displays it with the percentage in the sidebar.

Screenshot: 
Screenshot 2021-06-18 at 12 25 23

### CODE REVIEW

File: packages/app/src/domain/variants/variants-sidebar-metric.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+export function VariantsSidebarMetric({ data }: VariantsSidebarMetricProps) {
+  const { siteText, formatDateFromSeconds } = useIntl();
+  const commonText = siteText.common.metricKPI;
+
+  /**
+   * Filter all the keys that end with a _percentage.
+   */
+  const filteredKeysOnPercentage = Object.fromEntries(
+    Object.entries(data).filter(([key]) => key.endsWith('_percentage'))
+  ) as Pick<NlVariantsValue, `${Variant}_percentage`>;
```
Comment (VWSCoronaDashboard1) [inline packages/app/src/domain/variants/variants-sidebar-metric.tsx:21]: NICE! ;)

**Review by VWSCoronaDashboard1 — APPROVED** [review]

File: packages/app/src/domain/variants/variants-sidebar-metric.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+  /**
+   * Sort the keys from the highest to lowest and get the highest one
+   */
+  const highestPercentageVariant = Object.entries(
+    filteredKeysOnPercentage
+  ).sort((a, b) => b[1] - a[1])[0];
```
Comment (VWSCoronaDashboard10) [inline packages/app/src/domain/variants/variants-sidebar-metric.tsx:28]: could be simplified to:

```tsx
const highestPercentage = maxBy(
    Object.entries(data)
      .filter((x): x is [`${Variant}_percentage`, number] =>
        x[0].endsWith('_percentage')
      )
      .map(([key, value]) => ({
        variant: key.split('_')[0] as Variant,
        value,
      })),
    (x) => x.value
  );

// which in turn simplifies your jsx:

<InlineText fontSize={3} fontWeight="bold" margin="0" marginRight={3}>
  {`${highestPercentage.value}% ${
    siteText.covid_varianten.varianten[highestPercentage.variant]
  }`}
</InlineText>

**Review by VWSCoronaDashboard10 — APPROVED** [review]

File: packages/app/src/domain/variants/variants-sidebar-metric.tsx (hunk 2 above)
Comment (VWSCoronaDashboard10) [reply packages/app/src/domain/variants/variants-sidebar-metric.tsx:28]: or maybe even simpler:
```tsx
const highestPercentage = maxBy(
  Object.entries(data).map((x) =>
    x[0].endsWith('_percentage')
      ? {
          variant: x[0].split(')')[0] as Variant,
          value: x[1] as number,
        }
      : undefined
  ),
  (x) => x?.value
);

Comment (VWSCoronaDashboard14) [reply packages/app/src/domain/variants/variants-sidebar-metric.tsx:28]: 🥇

**Pull request merged**